doc/source/contributing/developer_guide.md

```diff
@@ -61,7 +60,7 @@ part of the docstring or from the _See also_ section.
 Sphinx is configured to ease referencing libraries ArviZ relies heavily on by
 using [intersphinx](https://docs.readthedocs.io/en/stable/guides/intersphinx.html).
 See guidance on the reference about how to link to objects from external
-libraries and the value of `intersphinx_mapping` in `conf.py` for the complete and up to
+libraries and the value of [intersphinx_mapping](https://github.com/arviz-devs/arviz/blob/main/doc/source/conf.py#L270) in [conf.py](https://github.com/arviz-devs/arviz/blob/main/doc/source/conf.py#L270) for the complete and up to
```

Comment:
Having links to the files on github is ok (I don't think it's necessary unless there could be some confusion on the file otherwise but it clearly doesn't hurt) however, I think we should avoid links with line numbers as they quickly stop pointing at the right place and end up being more confusing than having readers search for a variable or piece of code manually